webcompat/static/js/lib/bugform.js

```diff
@@ -142,7 +142,7 @@ function BugForm() {
     if (!location.search.search(/open=1/) && !location.search.search(/url=/)) {
       return;
     }
-    var urlParam = location.search.match(/url=(.+)/);
+    var urlParam = location.search.match(/url=([^&]*)/);
```

Comment:
(otherwise you ended up with stuff like URL: `http://example.com/&src=desktop-reporter`, etc.)